app/test_main.py

```diff
@@ -1,3 +1,35 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+class TestConvertAge:
+    @pytest.mark.parametrize(
+        "animals_age,into_human_age",
+        [
+            ((-2, -2), [0, 0]),
```

Comment:
In the video tutorials it is said that such tests are unnecessary, because there is an annotation of types, and if the developer sends the wrong type, Pycharm will send an error anyway

Reply:
Pycharm does not set any errors, just notifies developer that type is different than expected. You have request to check other types (string, list, float) in checklist as well